sw/device/lib/testing/spi_device_testutils.h

```diff
@@ -56,46 +56,249 @@ enum spi_device_command_slot {
   kSpiDeviceWriteCommandSlotBase = 11,
 };
 
+/*
+ * The SPI device pads for which `spi_device_testutils_configure_pad_attrs()`
+ * configures the pad attributes.
+ */
+static const top_earlgrey_direct_pads_t spi_device_direct_pads[4] = {
```

Comment:
Storage should not be defined in a header. Put them back in the C file.

Reply:
I think this is sorted now/